### PR TITLE
replace 777 by 775 and change file owner to seluser

### PR DESCRIPTION
**Pull request by eravion**
**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by diemol** [comment]
Can you explain why this is needed?

**Comment by eravion** [comment]
Hello @diemol ,
We have security check on docker folder rights, 777 are not allowed by our security dept..
This change fix this issue.
One border effect is managed by https://github.com/SeleniumHQ/docker-selenium/pull/1964

**Comment by VietND96** [comment]
Hi @eravion, may I know your security dept checks this compliance and reports it by which tool?

**Comment by eravion** [comment]
Hello @vietnd96 
Internal tool, sorry cannot share more.

**Comment by luisfcorreia** [comment]
This change seems logical to me.
Must investigate if there are any side effects

**Comment by eravion** [comment]
thanks @luisfcorreia 
Side effect identify during my test fixed by : https://github.com/SeleniumHQ/docker-selenium/pull/1964

File: Base/Dockerfile [hunk 1]
```diff
@@ -61,6 +61,8 @@ ENV HOME=/home/seluser
 # Add Grid check script
 #======================================
 COPY check-grid.sh entry_point.sh /opt/bin/
+# change right for /opt/bin*.sh file
+RUN chmod 775 /opt/bin/*.sh
```
Comment (diemol) [inline Base/Dockerfile:65]: Please do the `COPY` with `chmod` as mentioned in the Docker documentation https://docs.docker.com/engine/reference/builder/#copy

File: Base/Dockerfile [hunk 2]
```diff
@@ -72,7 +74,10 @@ COPY supervisord.conf /etc
 #==========
 RUN  mkdir -p /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
   && touch /opt/selenium/config.toml \
-  && chmod -R 777 /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor /etc/passwd \
+  # && chown seluser:seluser /var/run/supervisor /var/log/supervisor /opt/selenium /etc/passwd \
```
Comment (diemol) [inline Base/Dockerfile:77]: Why are you keeping this commented line?

File: Base/Dockerfile [hunk 3]
```diff
@@ -72,7 +74,10 @@ COPY supervisord.conf /etc
...
+  && chown seluser:seluser /var/run/supervisor /var/log/supervisor /opt/selenium \
```
Comment (diemol) [inline Base/Dockerfile:78]: Why did you remove `/etc/passwd`?

**Review by diemol — CHANGES_REQUESTED** [review]
Can you please check the comments and update your PR with the changes from `trunk`?

**Comment by VietND96** [comment]
Hi @eravion, `777` removal is already considered via PR #2056.
A new image tag will be released soon for your scan.
Thank you!

**Comment by VietND96** [comment]
Please use the new image tag `4.16.1-20231212` for your evaluation.
I will close the PR, feel free to continue the conversation around this feat #2056 or reopen this PR if something else you want to add. Thanks!

**Pull request closed without merging**